commands/operator-sdk/internal/cmd/generate/openapi.go

```diff
@@ -22,7 +22,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	genutil "github.com/operator-framework/operator-sdk/commands/operator-sdk/cmd/generate/internal"
+	genutil "github.com/operator-framework/operator-sdk/commands/operator-sdk/internal/cmd/generate/internal"
```

Comment:
We can probably move `genutil.go` from `generate/internal` to `generate` since now the entire `commands/operator-sdk/cmd` is internal.

/cc @estroz

Reply:
@AlexNPavel I don't think it hurts anything to have nested `internal` directory trees. If we leave `generate/internal` as is, it means that those functions cannot be used outside of the `generate` package. I don't have a strong opinion either way though.

@estroz Thoughts? Was your intent to hide those functions from end users or from other parts of the SDK?

Reply:
Seems like the functions are all used in the `generate` package so I agree moving them to the `internal/cmd/generate/genutil.go` makes sense, but I would also make them private, as at least from a quick search I can't seem to find them being used anywhere else but in that package.